app/views/tracks/index.html.haml

```diff
@@ -6,38 +6,74 @@
         %span.notranslate
           = @conference.title
 
+  .row
+    .col-md-12
+      %p.text-right
+        = link_to '#status-help', class: 'btn btn-default',  "data-toggle"=>"collapse" do
+          Help?
+      .collapse#status-help
+        %p
+          %strong
+            What happens next with my track request?
+        %p
+          If you submit a track request, the conference organizers will review it and either accept or reject it.
+          %br
+          If your track request is accepted, the conference organizers expect you to confirm that you will be able to hold it.
+          Then you will gain the Track organizer role.
+          %br
+          If your track request is rejected, you can either live with that or adapt it and resubmit it for review again.
```

Comment:
This is the same as in [proposals](https://github.com/openSUSE/osem/blob/master/app/views/proposals/index.html.haml#L40). I thought of it as kinda playful, so I kept it like that. If you insist, I will change it